optimade/server/routers/utils.py

```diff
@@ -78,7 +78,15 @@ def handle_response_fields(
     new_results = []
     while results:
         entry = results.pop(0)
-        new_entry = entry.dict(exclude_unset=True)
+
+        # TODO: re-enable exclude_unset when proper handling of known/unknown fields
+        # has been implemented
```

Comment:
Could you possibly reference the GH issue here?

Reply:
```suggestion
        # has been implemented (relevant issue: https://github.com/Materials-Consortia/optimade-python-tools/issues/263)
```